test/ingress/ingress.go

```diff
@@ -33,7 +33,7 @@ const (
 )
 
 // GetIngressEndpoint gets the ingress public IP or hostname.
-func GetIngressEndpoint(ctx context.Context, kubeClientset *kubernetes.Clientset) (string, error) {
+func GetIngressEndpoint(ctx context.Context, kubeClientset *kubernetes.Clientset, endpointOverride string) (string, func(string) string, error) {
```

Comment:
We want named returns of expanded comments here

Reply:
done